docs/architecture.md

```diff
@@ -3,7 +3,24 @@
 
 While we prefer in-code and live documentation over wiki pages, these are best suited to single services and we don't have a place to document broader processes that involve the interaction of several services. This section aims to provide that overview of how various services fit together.
 
-## Supporter data flow
+## Supporter Events Service
+
+The service aims to propagate supporter data to User Service and supporter events to ERP Service.
+
+User Service and ERP Service exist on one region only because of their structure; should they be unavailable for any reason, propagation of data from upstream services may be interrupted. Supporter Events is a multi-region service designed to handle incoming traffic from all sources. Events are queued to SQS and processed one by one.
+
+When identities are supplied to the `user-service` section of the payload, Supporter Events returns a salted hash of the identity that can be used by upstream services to identity resources, so that anonymisation won't break relationships between data and services.
+
+Supporter Service is supposed to handle only write requests to User Service and ERP Service. If an upstream service requires data stored in either service, it should either:
+
+- **For critical paths**: store a copy of the data in the service itself
```

Comment:
Are the `paths` defined somewhere? E.g. what is on the `critical` and what is on the `non-critical` path?

Reply:
Not at the moment

Reply:
There's a definition given just below saying that a non-critical path is one where "it is acceptable to present a failure to an end user". Would it work to move that up so that it's next to the first mention of critical/non-critical paths?

Reply:
👍🏻

Reply:
4ffd216817f068c20575913aab8f8dac65cee19f